<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkingTimeItem</class>
 <widget class="QWidget" name="WorkingTimeItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QDateTimeEdit" name="startDateTimeEdit">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;start time&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="calendarPopup">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateTimeEdit" name="endDateTimeEdit">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;end time&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="typeComboBox">
     <item>
      <property name="text">
       <string>working</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>pause</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>other</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
